### DISCUSSION TITLE
Add confirmation when sending vehicle group to depot

### DISCUSSION THREAD

**Discussion by wooky** [OpenTTD/OpenTTD#11132] [Features]
![image](https://github.com/OpenTTD/OpenTTD/assets/460371/c23c860a-823a-44b4-8045-c43f96b6947d)
You wouldn't believe the number of times I wanted to click Replace Vehicles, but I fat-fingered and pressed Send for Maintenance and paid the consequences. It would be nice to have a confirmation dialog popup when clicking Send for Maintenance/Send to Depot to avoid this problem.

**Comment by LC-Zorg** [comment]
Yep, click two pixels too low and you're dead. This is a nightmare if you have a really large network and hundreds or thousands of vehicles. Fixing all the mess that one accidental click can make can take more than an hour.

![Rozkaz wysłania do warsztatów - v1 1](https://github.com/OpenTTD/OpenTTD/assets/67662046/c3f6f0f0-1171-432e-b970-5062a2d4909d)
This is one of the first graphics and suggestions I wanted to propose (2019/01/27) ;)

**Comment by merni-ns** [comment]
Alternatively, JGRPP has a button to "Cancel servicing/send to depot" in the same menu. Maybe that could be backported?

**Reply by wooky** [reply]
I suppose that would be easier to implement, but from a player's perspective, the damage has already been done.

**Comment by DefinitelyNotRau117** [comment]
The simplest thing I can suggest in this situation is to add a separating line, as in the save-load-exit list. Not ideal, but still can help.

![изображение](https://github.com/OpenTTD/OpenTTD/assets/115672237/d81c63b4-dbb3-4eee-8f77-c26f91ed5469)
_(not implemented, just drawn in paint)_

**Comment by 2TallTyler** [comment]
#12284 attempts to remedy this with the divider line suggested by @DefinitelyNotRau117. 🙂

**Reply by 2TallTyler** [reply]
This has now been merged. It will not be in 14.0 as we are well past feature-freeze, but the nightly is available on Steam betas (and our website, of course) and 15.0 will be along before we know it. 🙂

**Reply by LC-Zorg** [reply]
Wouldn't such small but good changes be worth introducing in the meantime between major releases? :)
